static/index.html

```diff
@@ -6,8 +6,21 @@
   <title>Matoconv</title>
   <script>
 
-    function send()  {
-      var me_blob = new Blob([$('#content').val()], {type: 'text/html'});
+    function send(send_type)  {
+      var me_blob;
+      var content_type = 'text/html';
+      var file_name = 'test.html';
+      var file_data = $('#content').val();
+
+      if (send_type == 'file') {
+        var file = document.getElementById('upload_file').files[0];
```

Comment:
This variable is being pulled up to the top of the `send()` function scope.
Use `let` to force the scope to this if statement.
Or just put the definition at the top of the function to avoid any future confusion :)

On another note, given we've got jquery available and are using it above, this could be simplified to: `$('#upload_file').files[0]`